src/Formats/ProtobufSerializer.cpp

```diff
@@ -3341,6 +3345,7 @@ namespace
                 case TypeIndex::UInt256: return std::make_unique<ProtobufSerializerNumber<UInt256>>(column_name, field_descriptor, reader_or_writer);
                 case TypeIndex::Int8: return std::make_unique<ProtobufSerializerNumber<Int8>>(column_name, field_descriptor, reader_or_writer);
                 case TypeIndex::Int16: return std::make_unique<ProtobufSerializerNumber<Int16>>(column_name, field_descriptor, reader_or_writer);
+                case TypeIndex::Date32: return std::make_unique<ProtobufSerializerNumber<Int32>>(column_name, field_descriptor, reader_or_writer);
```

Comment:
Shouldn't this serializer be more elaborated? (see `ProtobufSerializerDateTime64`)

Reply:
Sure. Let me implement a separate serializer for it.

Reply:
Done